exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java

```diff
@@ -122,6 +125,31 @@ public String getQueryId() {
     return id;
   }
 
+  public String getPlanningDuration() {
+    //Check if Plan End is valid
+    if (profile.getStart() > profile.getPlanEnd()) {
```

Comment:
Code clarity is always a goal. One handy technique is to pick off conditions quickly. Here, we could do:
```
if (profile.getStart() <= profile.getPlanEnd()) {
  return ProfileResources.getPrettyDuration(profile.getStart(), profile.getPlanEnd());
}
// Don't estimate...